src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/features/form/data/submit/submitFormDataSagas.ts

```diff
@@ -84,18 +90,63 @@ function* submitFormSaga({ url, apiMode }: ISubmitDataAction): SagaIterator {
   }
 }
 
+function* saveFormDataSaga(): SagaIterator {
+  try {
+    const state: IRuntimeState = yield select();
+    const model = convertDataBindingToModel(state.formData.formData);
+    let validations = { ...state.formValidations.validations };
+    const componentSpecificValidations =
+      validateFormComponents(state.attachments.attachments, state.formLayout.layout, state.formData.formData,
+        state.language.language, state.formLayout.uiConfig.hiddenFields);
+    validations = Object.assign(validations, componentSpecificValidations);
+
+    const validationResult: IValidationResult = {
+      validations,
+      invalidDataTypes: state.formValidations.invalidDataTypes,
+    };
+
+    if (canFormBeSaved(validationResult, null)) {
```

Comment:
Errors detected by frontend would cause the form data not to be able to save?

Reply:
Only if the error is that the data type is incorrect (f.ex. submitting "abc" to an int field)